### ISSUE TITLE
How to revoke Veres One DID?

### ISSUE THREAD

**Issue by Drabiv** [w3c-ccg/didm-veres-one#2]
I cannot find in the specification how to revoke Veres One DID. Might be worth adding this.

**Comment by msporny** [comment]
Yes, good point, will add it to the spec during the next edit cycle.
